ruby/red-arrow-format/test/test-file-reader.rb

```diff
@@ -323,6 +339,154 @@ def test_read
                    ],
                    read)
     end
+
+    def test_type
+      assert_equal(:nanosecond, type.unit)
+    end
+  end
+
+  sub_test_case("Timestamp(:second)") do
+    def setup(&block)
+      @timestamp_2019_11_18_00_09_11 = 1574003351
+      @timestamp_2025_12_16_05_33_58 = 1765863238
```

Comment:
```suggestion
      @timestamp_2025_12_16_05_33_58 = 1765830838
```

```
[6] pry(main)> Time.new(2025,12,16,5,33,58).to_i
=> 1765830838
```

Reply:
This uses UTC. `Time.new` uses local time:

```console
$ ruby -e 'p Time.utc(2025,12,16,5,33,58).to_i'
1765863238
```